person/views/custom_client_view.xml

```diff
@@ -32,24 +30,15 @@
                 <field name="email" required="1"/>
                 <field name="photo" required="1"/>
                 <field name="gender" required="1"/>
+                <field name="postalcode"  required="1"/>
             </tree>
         </field>
     </record>
-
-
     <record id="action_client_example" model="ir.actions.act_window">
         <field name="name">Client name</field>
         <field name="res_model">custom.client</field>
         <field name="view_mode">tree,form</field>
     </record>
-
-    <menuitem id="menu_root_person"
-              name="Custom Client"/>
-
-    <menuitem id="menu_person_id1"
-              name="Client Details"
-              parent="menu_root_person"
-              action="action_client_example"/>
-
-
+    <menuitem id="menu_root_person" name="Custom Client"/>
```

Comment:
it was ok previously in column not row

 <menuitem id="menu_root_person"
              name="Custom Client"/>

    <menuitem id="menu_person_id1"
              name="Client Details"
              parent="menu_root_person"
              action="action_client_example"/>